hadoop-hdds/interface-client/src/main/proto/DatanodeClientProtocol.proto

```diff
@@ -108,6 +108,7 @@ enum Type {
 
   FinalizeBlock = 21;
   Echo = 22;
+  ReadBlock = 23;
```

Comment:
Please describe the command in the above comments section.

Reply:
done